packages/mapviewer/src/utils/__tests__/kmlUtils.spec.js

```diff
@@ -119,6 +121,18 @@ describe('Test KML utils', () => {
             ])
         })
     })
+    describe('isKmlFeaturesValid', () => {
+        it('returns false if there is an error in the features of the KML', () => {
+            const kml = readFileSync(resolve(__dirname, '../../../tests/cypress/fixtures/import-tool/kml_feature_error.kml'), 'utf8')
```

Comment:
It's not a bad idea to re-use assets from Cypress, I just fear it will be modified without knowing that it is used by a unit test too.

Would it be hard to generate quick "on-the-fly" fake KML data, such as in the `get extent of multiples features (marker, line, polygone)` test?

Or simply to copy this Cypress KML inside this `__test__` folder, so that its requirement is expressed in a clearer manner.